npm_tasks/build-core.js

```diff
@@ -0,0 +1,21 @@
+var fs = require('fs');
+var path = require('path');
+var execSync = require('child_process').execSync;
+
+var ROOT_DIR = path.resolve(__dirname, '..');
+var WEBMAKER_CORE_DIR = fs.realpathSync(path.resolve(
```

Comment:
For some reason this has to be the real path; `npm run build` seems to hang if we pass the unreal path as the `cwd`.